test/testutil/commandtests.go

```diff
@@ -105,7 +105,7 @@ func TableTestCommand(t *testing.T, commandTests CommandTests, newCommandFunc fu
 
 			// Assert mocks or other custom assertions
 			if tt.ExpectedAsserts != nil {
-				tt.ExpectedAsserts(t, clientsMock)
+				tt.ExpectedAsserts(t, ctxMock, clientsMock)
```

Comment:
note: Here's the nuts-and-bolts of this PR. We just pass the mock context into `ExpectedAsserts` so that the same context can be used everywhere. 🌟